### PR TITLE
Adds instructions to delete remote branch

### LINKED ISSUE #43: Add delete remote branch info
Deleting a remote branch is not done the same as for a local branch :

git push origin :myremotebranchtodelete

### PR DESCRIPTION
**Pull request by randomecho**
Syntax to delete remote branch is not the same as a local one.

Introduces the syntax mentioned in the book and
throws in the easier to remember one to boot.

Requested by, fixes #43

### CODE REVIEW

File: branching/index.html [hunk 1]
```diff
@@ -215,6 +215,36 @@
...
 </pre>
 
+    <h4>
+      git push (remote-name) :(branchname)
+      <small>delete a remote branch</small>
+    </h4>
+
+    <p>When you're done with a remote branch, whether it's been merged
+    into the remote master or you want to abandon it and sweep it under
+    the rug, you'll issue a <code>git push</code> command with special
```
Comment (matthewmccullough) [inline branching/index.html]: Perhaps some slight word tuning to "with a specially placed colon symbol"

and

"to remove that branch."

File: branching/index.html [hunk 2]
```diff
@@ -215,6 +215,36 @@
...
+    <code>git push remote-name local-branch:remote-branch</code> syntax.
+    This states that you want to push your local branch to match that
+    of the remote. When you remove the <code>local-branch</code> portion
+    you're now matching nothing to the remote, effectively telling the
+    remote branch to become nothing.
+    </p>
+
+    <p>Alternatively, you can also run
+    <code>git push remote-name --delete branchname</code>
+    which is basically a wrapper for the above colon prefix version.
```
Comment (matthewmccullough) [inline branching/index.html]: Remove word "basically"

append "...colon (refspec) technique of deleting a branch."

Comment (randomecho) [inline branching/index.html]: Expanded the "refspec" part since it looks like it's the first time it's mentioned in the guide.

**Comment by randomecho** [comment]
Would have been just the one commit with those changes, but forgot the part about refspec.

**Comment by matthewmccullough** [comment]
Very nice additions @randomecho

**Pull request merged**